client/my-sites/stats/stats-purchase/index.tsx

```diff
@@ -111,6 +111,7 @@ const StatsPurchasePage = ( { query }: { query: { redirect_uri: string; from: st
 							siteId={ siteId }
 							redirectUri={ query.redirect_uri ?? '' }
 							from={ query.from ?? '' }
+							disableFreeProduct={ isFreeOwned && ! isCommercialOwned && ! isPWYWOwned }
```

Comment:
```suggestion
							disableFreeProduct={ isFreeOwned || isCommercialOwned || isPWYWOwned }
```

Should we also disable the free product when users have purchased any Stats products since Stats free is the lowerest level?

Reply:
I agree! Thanks for the feedback.